tests/components/apsystems/test_number.py

```diff
@@ -44,7 +46,11 @@ async def test_all_entities(
     snapshot: SnapshotAssertion,
     mock_config_entry: MockConfigEntry,
     entity_registry: er.EntityRegistry,
+    freezer: FrozenDateTimeFactory,
 ) -> None:
     """Test all entities."""
     await setup_integration(hass, mock_config_entry)
+    freezer.tick(datetime.timedelta(seconds=30))
+    async_fire_time_changed(hass)
+    await hass.async_block_till_done()
```

Comment:
Looks like this entity doesn't update, when added. I can implement 
`add_entities([ApSystemsMaxOutputNumber(config_entry.runtime_data)], update_before_add=True)`
In a separate PR?

Reply:
I think it's okay, to do it in this PR, when renaming it.